### ISSUE TITLE
Use Libhandy

### ISSUE THREAD

**Issue by airon90** [sharkwouter/minigalaxy#18]
I suggest you to use [libhandy](https://source.puri.sm/Librem5/libhandy) so that your app can fit also in small screens

**Comment by sharkwouter** [comment]
This is an interesting suggestion. Do you have a screen which has an issue witht he current interface? Could you post a screenshot?

I'm not sure if libhandy would be easy to implement or if it would be worthwhile. The current interface can be resized to a width of about 600 pixels without causing the functionality to be limited.

**Comment by airon90** [comment]
> Do you have a screen which has an issue witht he current interface?

I don't have a small screen to test your app. I just resized it and I saw that it would not fit into small screen. E.g. header bar and dialogues are not adapting when resized. If using a PC is not that important but if using small screens such as smartphone (Librem 5) or small RPi screens it could be useful. 

> I'm not sure if libhandy would be easy to implement or if it would be worthwhile

AFAIK it is easy to implement, you will decide if worthwhile. For sure it is just a suggestion to implement such function, I don't think it is so important to solve it now :)
